pkg/version/version_data.go

```diff
@@ -57,6 +57,11 @@ type StableVersion struct {
 // LoadStableVersion loads the stable version data from the version configuration directory returning an empty object if there is
 // no specific stable version configuration available
 func LoadStableVersion(wrkDir string, kind VersionKind, name string) (*StableVersion, error) {
+	if filepath.IsAbs(name) {
```

Comment:
this should be the thing that allows us to install correctly - however it is impossible to test at the moment (on either  OS-X or windows) as the vault installation is failing.